gamecenter/public/views.py

```diff
@@ -1,17 +1,69 @@
 # -*- coding: utf-8 -*-
 """Public section, including homepage and signup."""
-from flask import (Blueprint, render_template)
+from flask import (
+    Blueprint,
+    flash,
+    redirect,
+    render_template,
+    request,
+    url_for,
+)
+from flask_login import login_user, login_required, logout_user
+
+from gamecenter.api.views import create_game
+from gamecenter.extensions import login_manager
+from gamecenter.public.forms import LoginForm
+from gamecenter.user.forms import RegisterForm
+from gamecenter.user.models import User
+from gamecenter.utils import flash_errors
 
 blueprint = Blueprint('public', __name__, static_folder="../static")
 
 
-@blueprint.route("/")
+@login_manager.user_loader
+def load_user(id):
+    return User.get_by_id(int(id))
+
+
+@blueprint.route("/", methods=['GET', 'POST'])
 def home():
     """Return the home page"""
-    return render_template("public/home.html")
+    form = LoginForm(request.form)
+    if request.method == 'POST':
+        if form.validate_on_submit():
+            login_user(form.user)
+            redirect_url = request.args.get("next") or url_for("user.dashboard")
+            return redirect(redirect_url)
+        else:
+            flash_errors(form)
+    return render_template("public/home.html", form=form)
+
+
+@blueprint.route('/logout/')
```

Comment:
no, it's just the same format as the cookiecutter had, I can remove the trailing slash if that bothers you